framework/Samplers/Sampler.py

```diff
@@ -41,46 +41,10 @@
 class Sampler(utils.metaclass_insert(abc.ABCMeta,BaseType),Assembler):
   """
     This is the base class for samplers
-    Samplers own the sampling strategy (Type) and they generate the
```

Comment:
why did you remove this?

Reply:
It has not been maintained and has many inaccuracies, and largely replaced by the docstrings.  If we really do want to maintain something like this, I think we need to require PRs review it for changes, and make it a general requirement throughout RAVEN.